src/components/Match/MatchStory.jsx

```diff
@@ -0,0 +1,543 @@
+/* global API_HOST */
+import React from 'react';
+import strings from 'lang';
+import {
+  formatSeconds,
+  jsonFn,
+} from 'utility';
+import { IconRadiant, IconDire } from 'components/Icons';
+import heroes from 'dotaconstants/build/heroes.json';
+import items from 'dotaconstants/build/items.json';
+import styles from './Match.css';
+
+const heroesArr = jsonFn(heroes);
+const radiantColor = '#66bb6a';
+const direColor = '#ff4c4c';
```

Comment:
Whoops, you can use `styles.red` and `styles.green` and any of exported colors in `palette.css` that is imported in 'Match.css'